src/main/resources/db/migration/V41__rettet_update_vilkar_enums.sql

```diff
@@ -0,0 +1 @@
+UPDATE behandling SET steg = 'VILKÅR' WHERE steg in ('REGISTRERE_OPPLYSNINGER', 'VILKÅRSVURDERE_INNGANGSVILKÅR', 'VILKÅRSVURDERE_STØNAD');
```

Comment:
Vi har allerede renamet verdiene til VILKAR så denne må være:
UPDATE behandling SET steg = 'VILKÅR' WHERE steg in ('VILKAR');